components/member-profile/index.js

```diff
@@ -6,6 +6,7 @@ import classNames from './member-profile.module.scss';
 import ContributionType from './contribution-type/';
 import { motion } from 'framer-motion';
 import Modal from 'components/modal';
+import Showskills from '../member-card/showskills';
```

Comment:
please make use of absolute imports.
`components/member-card/show-skills`

Reply:
Now using absolute imports.